### ISSUE TITLE
OLD-6754: records being saved in eavvaluesvarchar have wrong eaventity_type

### ISSUE THREAD

**Issue by cpaschen** [dioscouri/tienda#38]
## Description

In testing with a large number of EAV Custom Fields, all of which are associated with products, the system is saving many records with the type "carts" instead of "orderitems".
###### Updated by Chris DioscouriChris Paschen. 8 months ago

Target version changed from Version 0.9.9 - Misc Bug Fixes and New Features to Version 0.9.1 - EAV (Custom Fields) Finalization
###### #1 Updated by Chris DioscouriChris Paschen. 8 months ago

It appears that the entries that are assigned to the 'cart' entity_type are ones that are associated with an eaventity_id (and product) where that specific entity shouldn't be applied (i.e. the custom field should not be applied to that product).

And it's retaining content from a previously entered field.

However, in reviewing the contents of the eavValuesVarChar, I can't 100% confirm this as I can't completely follow how the data is stored in this file.

It appears that the eaventity_id references the orderitems ID field; however, there are custom fields being displayed (on the orders screen) that are not in this table.

Bottom line ... I think we need a short explanation on how the data is stored in the EAV (how it works) and what fields reference what tables. (A data diagram would be VERY handy about now)

**Comment by rdiaztushman** [comment]
In my testing the past few weeks, I haven't been able to reproduce this bug
